charts/cluster-api-runtime-extensions-nutanix/defaultclusterclasses/aws-cluster-class.yaml

```diff
@@ -108,6 +108,33 @@ spec:
             extraArgs:
               profiling: "false"
         files:
+        - content: |
```

Comment:
Its easiest to just look at the output in the ClusterClasses and see how its equivalent to what we had before.